test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -108,11 +108,13 @@ public string[] GetInstallCommandsFromLocal(string path)
         {
             SupportedPackageExtension.Deb => new[]
             {
-                "apt-get purge --yes aziot-edge aziot-identity-service libiothsm-std iotedge"
+                "apt-get purge --yes aziot-edge aziot-identity-service libiothsm-std iotedge",
```

Comment:
Since this is a `release/1.1` branch. The removal of `aziot-*` are not necessary -- it doesn't hurt to have them as a precaution. If you want to keep the `aziot` removal here, please use `aziot-*` instead of specifying the name 2/4 of the existing `aziot` components.

Reply:
I dont think I have made any changes to that line of code , the only change was to add the restart docker command.

Reply:
> Would you want me to make this change in this PR or create a separate PBI to update this?

Yes please.

Reply:
Created PBI : https://dev.azure.com/msazure/One/_workitems/edit/11028682